docs/troubleshoot/app-framework/appserver-error-codes.md

```diff
@@ -1800,291 +1800,348 @@ The following error message codes may appear on the app-server log. Use the foll
 
   **Action:**
 
-  TODO
+  Check if your war file is in a valid format or exists.
 
 
 
-### ZWED0078W":"Could not access files to determine status for service=%s, error=
+### ZWED0078W
+
+  Could not access files to determine status for service=_key-value_, error=_errorMessage_
 
   **Reason:**
 
-  TODO
+  If we are unable to get the status of war extracted or not, then it throws _errorMessage_ in catch block.
 
   **Action:**
 
-  TODO
+  Check if your war file is in a valid format or not.
+
 
 
+### ZWED0079W
 
-### ZWED0079W":"Cannot add servlet for service=%s, error=
+  Cannot add servlet for service=_key-value_, error=_errorMessage_
 
   **Reason:**
 
-  TODO
+  If unable to add servlet for service _key-value_, then it logs a warning _errorMessage_.
 
   **Action:**
 
-  TODO
+  No action is needed, but the warning may be needed to debug
+
 
 
+### ZWED0080W
 
-### ZWED0080W":"Cannot add servlet for service=%s
+  Cannot add servlet for service=_key-value_
 
   **Reason:**
 
-  TODO
+  When we are not able to get the _dir_ to add servlet for service _key-value_.
 
   **Action:**
 
-  TODO
+  Check if your _dir_ is exists / in valid format or not.
 
 
 
-### ZWED0081W":"Could not start tomcat, error=
+### ZWED0081W
+
+  Could not start tomcat, error=_errorMessage_
 
   **Reason:**
 
-  TODO
+  If tomcat manager is unable to start tomcat with _DOptionsArray_, then it logs _errorMessage_ in catch block.
 
   **Action:**
 
-  TODO
+  Review the warning message and verify _DOptionsArray_ element values are valid or not.
+
 
 
+### ZWED0082W
 
-### ZWED0082W":"%s stderr=%s
+  _this.getIdString()_ stderr=_data_
 
   **Reason:**
 
-  TODO
+  A tomcat process with _this.getIdString()_ encountered an standard error with receiving _data_.
 
   **Action:**
 
-  TODO
+  Action depends on what _data_ is. May be useful in debugging.
+
 
 
+### ZWED0083W
 
-### ZWED0083W":"Tomcat could not start. Closing. code=%s
+  Tomcat could not start. Closing. code=_code_
 
   **Reason:**
 
-  TODO
+  If tomcat manager is unable to start Tomcat, and closing the tomcat process with _code_
+  then it logs this message.
 
   **Action:**
 
-  TODO
+  Review the warning message and see if you are able to check why the tomcat unable to start.
 
 
 
-### ZWED0084W":"Tomcat could not start. Exiting. code=%s
+### ZWED0084W
+
+  Tomcat could not start. Exiting. code=_code_
 
   **Reason:**
 
-  TODO
+  If tomcat manager is unable to start Tomcat, and exiting from the tomcat process
+  with _code_ then it logs this message.
 
   **Action:**
 
-  TODO
+  Review the warning message and see if you are able to check why the tomcat unable to start
+  and exited from tomcat process.
+
 
 
+### ZWED0085
 
-### ZWED0085W":"%s Error when stopping, error=
+  _getIdString()_ Error when stopping, error=_errorMessage_
 
   **Reason:**
 
-  TODO
+  An error occurred while stopping tomcat-process for windows then it logs _errorMessage_ with _getIdString()_.
```

Comment:
Similar as my comment: https://github.com/zowe/docs-site/pull/2526/files#r1038322812

This one I'll leave for you to figure out what it should be!